glean-core/ffi/src/handlemap_ext.rs

```diff
@@ -41,7 +41,7 @@ pub fn log_if_error(error: ExternError) {
     if !error.get_code().is_success() {
         // in practice this should never panic, but you never know...
         ffi_support::abort_on_panic::call_with_output(|| {
-            log::error!(
+            log::warn!(
```

Comment:
IMO this can stay an error. It's an exceptional case and we better know about this early (even if it occasionally results in noise we can't track down)

```suggestion
            log::error!(
```